src/lib/renderComponent.tsx

```diff
@@ -21,15 +24,33 @@ export interface IRenderConfig {
   displayName?: string
   handledProps: string[]
   props: { [key: string]: any }
+  state: { [key: string]: any }
   rules?: { [key: string]: Function }
   variables?: (siteVariables: object) => object
 }
 
+const getAccessibility = <P extends {}>(props, state) => {
+  let accessibility = props['accessibility'] || props['defaultAccessibility'] || DefaultBehavior
+  if (typeof accessibility === 'function') {
```

Comment:
we have `callable` monad for these purposes. Just as a reference, in the same file for variables:

```js
const variablesFromFile = callable(variables)(siteVariables)
const variablesFromTheme = callable(componentVariables[displayName])(siteVariables)
const variablesFromProp = callable(props.variables)(siteVariables)
```

This function can be rewritten in the following way:

```js
getAccessibility() {
   const { accessibility: customAccessibility, defaultAccessibility } = props
   return callable(customAccessibility || defaultAccessibility ||  DefaultBehavior)(...props, ...state)
}
```

```